packages/modeling/src/geometries/poly2/isA.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Determine if the given object is a 2D polygon.
+ * @param {Object} object - the object to interrogate
+ * @returns {Boolean} true if the object matches a poly2
+ * @alias module:modeling/geometries/poly2.isA
+ */
+const isA = (object) => {
+  if (object && typeof object === 'object') {
+    if ('vertices' in object) {
```

Comment:
The if statement on line 9 is unecessary because the check on line 10 covers the case where object.vertices is undefined.

Reply:
Also, could simplify line 9 and 10 into:

```
  return Array.isArray(object.vertices)
```